doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -95,7 +95,9 @@ Security
 ========
 
   * Added support for HKDF-Expand and HKDF-Extract in CRACEN.
-  * Added support for Ed25519ph(HashEdDSA) to CRACEN
+  * Added support for Ed25519ph(HashEdDSA) to CRACEN.
```

Comment:
* Added:

  * Support for HKDF-Expand and HKDF-Extract in CRACEN.
  * Support for Ed25519ph(HashEdDSA) to CRACEN.